envs/.env.dev

```diff
@@ -10,6 +10,11 @@ NEXT_PUBLIC_BUILD_TYPE=vagovdev
 # Location to load vets-website assets
 NEXT_PUBLIC_ASSETS_URL=https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/
 
+# Which loggers to show messages from in the console.
+# https://www.npmjs.com/package/debug
+DEBUG='*,-eslint,-eslint*,-typescript*,-next:jsconfig-paths-plugin,-socks-proxy-agent'
+DEBUG_COLORS=true
```

Comment:
This maybe shouldn't be `true`? 🤔 🤷

I've set it to `true` in the other environments because the output should be a TTY, but since most of the work happens in a child worker, by default, it doesn't use colors. See https://www.npmjs.com/package/debug#usage-in-child-processes. I'm not sure how this will affect the output in CI, or what to do about that if it's ugly. 🙈